### PR TITLE
Open imgur upload in screen center

### LINKED ISSUE #806: Imgur save window never appears in the screen center
Captura de tela_2020-08-26_01-11-20

The 'upload to imgur' window never appears in the center of the screen, so I have to drag it up. Is there a way to make it appear on the center or top of screen? Would be awesome to make it just upload to imgur and copy the link to clipboard without buttons or a window.

Thank you so much.

### PR DESCRIPTION
**Pull request by AlexP11223**
Fixes #806

The same code as in `InfoWindow`.

### CODE REVIEW

File: src/tools/imgur/imguruploader.cpp [hunk 1]
```diff
@@ -48,6 +52,13 @@ ImgurUploader::ImgurUploader(const QPixmap& capture, QWidget* parent)
     setWindowTitle(tr("Upload to Imgur"));
     setWindowIcon(QIcon(":img/app/flameshot.svg"));
 
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 10, 0))
```
Comment (borgmanJeremy) [inline src/tools/imgur/imguruploader.cpp:55]: Do you know if there is an alternative API that supports QT 5.9?

Comment (AlexP11223) [reply src/tools/imgur/imguruploader.cpp:55]: I am not sure which part of this is not supported in 5.9 :D

Comment (borgmanJeremy) [reply src/tools/imgur/imguruploader.cpp:55]: Why is there a QT version check macro?

Comment (AlexP11223) [reply src/tools/imgur/imguruploader.cpp:55]: ah, it's `screenAt` https://github.com/flameshot-org/flameshot/pull/318#issue-209315899

**Pull request merged**